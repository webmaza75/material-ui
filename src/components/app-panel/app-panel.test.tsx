import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import AppPanel from './app-panel';

describe(`AppPanel`, () => {
  it(`AppPanel renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const selected = [];
    const result = renderer
      .render(<AppPanel
        selected={selected}
        onSelectionReset={jest.fn()}
        onItemsDelete={jest.fn()}
        searchTitle={''}
        onInputChange={jest.fn()}
        isOpenConfirmDeleteDialog={false}
        onTasksCancelDelete={jest.fn()}
        onTasksConfirmDelete={jest.fn()}
        isOpenUndoDeleteSnackbar={false}
        onItemsExactlyDelete={jest.fn()}
        onItemsUndoDelete={jest.fn()}
      />);
    
    expect(result).toMatchSnapshot();
  });
});
