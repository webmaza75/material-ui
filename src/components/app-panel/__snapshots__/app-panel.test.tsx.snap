// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppPanel AppPanel renders correctly 1`] = `
<ForwardRef(WithStyles)
  className="makeStyles-appBar-4"
>
  <ForwardRef(WithStyles)
    className="makeStyles-toolbar-2"
  >
    <ForwardRef(WithStyles)
      aria-label="Menu"
      color="inherit"
      edge="start"
    />
    <ForwardRef(WithStyles)
      aria-label="Add"
      className="makeStyles-fabButton-8"
      color="secondary"
    >
      <Memo />
    </ForwardRef(WithStyles)>
    <div
      className="makeStyles-wrapper-12"
    >
      <ForwardRef(WithStyles)
        className="makeStyles-subTitle-3"
        variant="h6"
      >
        Automated Tasks
      </ForwardRef(WithStyles)>
      <div
        className="makeStyles-search-6"
      >
        <ForwardRef(WithStyles)
          alignItems="flex-end"
          container={true}
          spacing={1}
        >
          <ForwardRef(WithStyles)
            item={true}
          >
            <Memo
              className="makeStyles-searchIcon-7"
            />
          </ForwardRef(WithStyles)>
          <ForwardRef(WithStyles)
            item={true}
          >
            <ForwardRef(WithStyles)
              InputProps={
                Object {
                  "classes": Object {
                    "input": "makeStyles-inputField-10",
                    "underline": "makeStyles-underline-11",
                  },
                }
              }
              className="makeStyles-textField-9"
              id="standard-search"
              margin="normal"
              onChange={[MockFunction]}
              type="search"
              value=""
            />
          </ForwardRef(WithStyles)>
        </ForwardRef(WithStyles)>
      </div>
    </div>
  </ForwardRef(WithStyles)>
  <ConfirmationDeleteDialog
    onTasksCancelDelete={[MockFunction]}
    onTasksConfirmDelete={[MockFunction]}
    open={false}
  />
  <SimpleSnackbar
    isOpenUndoDeleteSnackbar={false}
    onItemsExactlyDelete={[MockFunction]}
    onItemsUndoDelete={[MockFunction]}
  />
</ForwardRef(WithStyles)>
`;
